packages/migrate/src/__tests__/MigrateResolve.test.ts

```diff
@@ -203,6 +203,7 @@ describeIf(!process.env.TEST_SKIP_COCKROACHDB)('cockroachdb', () => {
           `)
 
     expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+      Environment variables loaded from prisma/.env
```

Comment:
What causes this unrelated change?

Reply:
The fact that `packages/migrate/src/__tests__/fixtures/schema-only-cockroachdb/prisma` contains a `.env` file, which is automatically load by the [`loadEnv`](https://github.com/prisma/prisma/blob/c1d4f6ea4a0499759dcc1e43cc6c846ccb8051a8/packages/sdk/src/utils/tryLoadEnvs.ts#L119-L121) function.

Reply:
Ah, and before the `schema-only-cockroachdb` test fixture did not - you added that 👍